examples/AeroSharp.Examples/Maps/NestedCDTMapGetByRankExample.cs

```diff
@@ -0,0 +1,57 @@
+using AeroSharp.Connection;
+using AeroSharp.DataAccess;
+using AeroSharp.DataAccess.MapAccess;
+using AeroSharp.Examples.Utilities;
+using Aerospike.Client;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AeroSharp.Examples.Maps;
+
+public class NestedCdtMapGetByRankExample : IExample
```

Comment:
I think it's going to be really important to comment what this example does and means. I know you've got this in your head now, but think about you six months from now. Also, this might be a little hard to comprehend for those less familiar with maps. This is probably true for the other examples as well.